airbyte/destinations/util.py

```
@@ -56,6 +57,9 @@ def get_destination(  # noqa: PLR0913 # Too many arguments
             the host network. This is useful for connectors that need to access resources on
             the host machine, such as a local database. This parameter is ignored when
             `docker_image` is not set.
+        host_temp_dir: If set, along with docker_image, this replaces the volume exposing the temporary
```

Ruff (0.8.2)

Line too long (103 > 100)

(E501)